### ISSUE TITLE
A issue for cityscapes dataset

### ISSUE THREAD

**Issue by qixuxiang** [Eromera/erfnet_pytorch#3]
Hi , Thx your code , I am very interested in your work .
But I don't know exactly which dataset I need . There is so many lable in Cityscapes dataset.
I guess the dataset is leftImg8bit_trainvaltest.zip (11GB) and the corresponding ground truth gtFine_trainvaltest.zip (241MB),haha.
And my computer do not has a GPU,which has a poor performance.
It is indeed double happiness for me to have  a pretrained net , which contains txt or xml that compute before. Thanks a lot!

**Comment by Eromera** [comment]
Hi @qixuxiang, thanks for your message! 

You are correct, the dataset imgs and labels used in the current code are the leftImg8bit_trainvaltest.zip (11GB) and gtFine_trainvaltest.zip (241MB) from Cityscapes, although you could change the code to use any other datasets.

The code is able to run fully in CPU (although you would have to change the code slightly) without a GPU, but it would take very long time to train so this is not very recommended. Running some evaluation scripts to use the model without training should be ok in CPU if you are ok with the additional time that it takes compared to GPU.

The model provided in "trained_models/erfnet_pretrained.pth" is already trained in cityscapes train set and has an accuracy of 72.2% IoU in the Val set. You can use this for evaluation. I did not understand your last part about the txt or xml file, if you meant about the weights saved in a file then these are saved in the .pth file that I mentioned.

Hope I could help!

**Comment by qixuxiang** [comment]
Thanks for your reply .
yeah, I meant about the weights saved in a file, The file in "trained_models/erfnet_pretrained.pth" is what I need. I am going to check the file for evaluation.
I‘m much obliged to you, Have a nice day! 
(∩_∩)

**Comment by Eromera** [comment]
No problem! The ".pth" file was saved by PyTorch's "torch.save" function, so it's using their standard which is a "dictionary" containing the names of each layer and the associated weights. It can be loaded with "torch.load()". The scripts in the "erfnet_pytorch/eval/" folder load this file so you might check them for code examples. 

I will close the issue, please re-open it if you have more doubts. Have a nice day you too!
